testsuite/MDAnalysisTests/analysis/test_wbridge.py

```diff
@@ -723,3 +723,26 @@ def test_timesteps_by_type(self, wb_multiframe):
 
         timesteps = sorted(wb_multiframe.timesteps_by_type())
         assert_array_equal(timesteps[3], [1, 12, 'ALA', 1, 'H', 'ALA', 6, 'O', 0, 2])
+
+    def test_duplicate_water(self):
+        '''A case #3119 where
+        Acceptor···H−O···H-Donor
+                     |
+                     H···O-H
+        will be recognised as 3rd order water bridge.
```

Comment:
The matching issue has "Will be recognised as a third-order water bridge"  under the "expected behavior" section?

Is this supposed to be a "first order" water bridge because only one water actually contacts both the acceptor an donor? Or is it second order because of propagation into the "extra water" on the side?

Reply:
I'm sorry. I didn't realise that it is in the expected behaviour section. I have amended the issue to reflect that.